src/diffusers/models/attention.py

```diff
@@ -252,7 +248,8 @@ def __init__(
                 ada_norm_bias,
                 "layer_norm",
             )
-        elif not self.use_ada_layer_norm_single:
+
+        elif norm_type in ["ada_norm_zero", "ada_norm", "layer_norm", "ada_norm_continuous"]:
             self.norm3 = nn.LayerNorm(dim, norm_eps, norm_elementwise_affine)
```

Comment:
I think it is better to still have `norm3` but set to `None`
```suggestion
            self.norm3 = nn.LayerNorm(dim, norm_eps, norm_elementwise_affine)
     elif norm_type == "layer_norm_i2vgen":
            self.norm3 = None
```

Reply:
Sweet!